plugins/tff_backend/consts/kyc.py

```diff
@@ -0,0 +1,206 @@
+        'keyboard_type': 'PHONE',
+        'order': 8
+    },
+    {
+        'type': 'Communication',
+        'sub_type': 'EmailAddress',
+        'message': 'Please enter your email address',
+        'keyboard_type': 'EMAIL',
+        'order': 9
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'StreetName',
+        'message': 'Please enter the street name of your home address',
+        'order': 10
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'BuildingNumber',
+        'message': 'Please enter the building number of your home address',
+        'order': 11
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'BuildingName',
+        'message': 'Please enter the name of the building of your home address',
+        'order': 12
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'UnitNumber',
+        'message': 'Please enter the flat/unit/apartment number of your home address',
+        'order': 13
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'StreetType',
+        'message': 'Please enter the street type of your home address (Typically St, Rd, etc.)',
+        'order': 14
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'Suburb',
+        'message': 'Please enter the suburb / subdivision / municipality of your home address',
+        'order': 15
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'POBox',
+        'message': 'Please enter your Post Office Box number',
+        'order': 16
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'City',
+        'message': 'Please enter the city of your home address',
+        'order': 17
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'Country',
+        'message': 'Please enter the country of your home address.',
+        'widget': 'SelectSingleWidget',
+        'choices': country_choices,
+        'order': 18
+    },
+    {
+        'type': 'Location',
+        'sub_type': 'PostalCode',
+        'message': 'Please enter the postal code of your home address.',
+        'order': 19
+    },
+    {
+        'type': 'DriverLicense',
+        'sub_type': 'Number',
+        'message': 'Please enter your Driver\'s License Number',
+        'order': 20
+    },
+    {
+        'type': 'DriverLicense',
+        'sub_type': 'State',
+        'message': 'Please enter the state of issue for your Driver\'s License',
+        'order': 21
+    },
+    {
+        'type': 'DriverLicense',
+        'sub_type': 'DayOfExpiry',
+        'message': 'Please select the expiry date of your Driver\'s License',
+        'widget': 'SelectDateWidget',
+        'order': 22
+    },
+    {
+        'type': 'NationalIds',
+        'sub_type': 'Number',
+        'message': 'Please enter your national id number',
+        'order': 23
+    },
+    {
+        'type': 'NationalIds',
+        'sub_type': 'CityOfIssue',
+        'message': 'Please enter the distinct that issued the ID',
```

Comment:
I would say 'Please enter the distinct that issued **your national ID**'
And is distinct the correct word? I dont know it in this context. I would have expected "city" or "district"?